src/views/SelectJobsModal.vue

```diff
@@ -10,9 +10,13 @@
     </ion-toolbar>
   </ion-header>
   <ion-content>
-    <ion-searchbar v-model="queryString" :placeholder="$t('Search jobs')" @keyup.enter="search($event)" />
+    <ion-searchbar :placeholder="$t('Search jobs')" @keyup.enter="search($event)" />
 
-    <div v-if="jobs.length === 0" class="ion-text-center">
+    <div v-if="queryString.length === 0" class="ion-text-center">
+      <p>{{ $t("Searched jobs will appear here") }}</p>
+    </div>    
+
+    <div v-else-if="jobs.length === 0 && queryString.length != 0" class="ion-text-center">
```

Comment:
I think the condition `queryString.length != 0"` might not be needed, else-if will execute only when `queryString.length === 0` fails.

Reply:
Improved.